_overviews/cheatsheets/index.md

```diff
@@ -26,270 +26,271 @@ permalink: /cheatsheets/index.html
-      <td>define function <br /> hidden error: without = it’s a Unit-returning procedure; causes havoc</td>
+      <td>Define function.<br />Hidden error: without <code>=</code> it’s a <code>Unit</code>-returning procedure; causes havoc.</td>
     </tr>
     <tr>
       <td><span class="label success">Good</span><br> <pre class="highlight"><code>def f(x: Any) = println(x)</code></pre><br /> <span class="label important">Bad</span><br> <pre class="highlight"><code>def f(x) = println(x)</code></pre></td>
-      <td>define function <br /> syntax error: need types for every arg.</td>
+      <td>Define function.<br />Syntax error: need types for every arg.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>type R = Double</code></pre></td>
-      <td>type alias</td>
+      <td>Type alias.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>def f(x: R)</code></pre> vs.<br /> <pre class="highlight"><code>def f(x: =&gt; R)</code></pre></td>
-      <td>call-by-value <br /> call-by-name (lazy parameters)</td>
+      <td>Call-by-value.<br /><br />Call-by-name (lazy parameters).</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>(x:R) =&gt; x * x</code></pre></td>
-      <td>anonymous function</td>
+      <td><pre class="highlight"><code>(x: R) =&gt; x * x</code></pre></td>
+      <td>Anonymous function.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>(1 to 5).map(_ * 2)</code></pre> vs.<br /> <pre class="highlight"><code>(1 to 5).reduceLeft( _ + _ )</code></pre></td>
-      <td>anonymous function: underscore is positionally matched arg.</td>
+      <td><pre class="highlight"><code>(1 to 5).map(_ * 2)</code></pre> vs.<br /> <pre class="highlight"><code>(1 to 5).reduceLeft(_ + _)</code></pre></td>
+      <td>Anonymous function: underscore is positionally matched arg.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>(1 to 5).map( x =&gt; x * x )</code></pre></td>
-      <td>anonymous function: to use an arg twice, have to name it.</td>
+      <td><pre class="highlight"><code>(1 to 5).map(x =&gt; x * x)</code></pre></td>
+      <td>Anonymous function: to use an arg twice, have to name it.</td>
     </tr>
     <tr>
       <td><span class="label success">Good</span><br> <pre class="highlight"><code>(1 to 5).map(2 *)</code></pre><br /> <span class="label important">Bad</span><br> <pre class="highlight"><code>(1 to 5).map(* 2)</code></pre></td>
-      <td>anonymous function: bound infix method.<br /> Use <code class="highlighter-rouge">2 * _</code> for sanity’s sake instead.</td>
+      <td>Anonymous function: bound infix method.<br /> Use <code class="highlighter-rouge">2 * _</code> for sanity’s sake instead.<br />Will require <code>import scala.language.postfixOps</code> in the future.</td>
```

Comment:
In light of https://github.com/scala/scala/pull/6559 maybe we just delete this row from the page?  It doesn't seem relevant for a cheatsheet, and post fix ops are on their way out.